kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/wlst/createWLSDomain.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+#
+#Copyright (c) 2021, Oracle and/or its affiliates.
+#Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+#Adopted from https://github.com/oracle/docker-images/blob/main/OracleWebLogic/samples/12213-domain-home-in-image/container-scripts/createWLSDomain.sh
+#Define DOMAIN_HOME
+echo "Domain Home is: " $DOMAIN_HOME
+
+ADD_DOMAIN=1
+if [  -f ${DOMAIN_HOME}/servers/${ADMIN_NAME}/logs/${ADMIN_NAME}.log ]; then
+    exit
+fi
+
+# Create Domain only if 1st execution
+DOMAIN_PROPERTIES_FILE=${PROPERTIES_FILE_DIR}/domain.properties
+if [ ! -e "${DOMAIN_PROPERTIES_FILE}" ]; then
+   echo "A properties file with the username and password needs to be supplied."
```

Comment:
Maybe make the error message a little more helpful? For example:
"Error: Property file '${DOMAIN_PROPERTIES_FILE}' not found. A properties file with the username and password needs to be supplied."

Reply:
done